src/pmx/scripts/analyze_dhdl.py

```diff
@@ -608,6 +608,9 @@ def main(args):
                                                                                  p=prec, u=units), quiet=quiet)
 
     _tee(out, ' ========================================================', quiet=quiet)
+    
+    #close the outputfile, forsing a fush to disk, so that the unit test can read it properly
+    out.close();
```

Comment:
semicolon

Reply:
Any reason to add semicolons to just those lines?
I was going off of the [google style guide for python on semicolons](http://google.github.io/styleguide/pyguide.html#31-semicolons).

Reply:
ah, right, it is a _removed_ semicolon. That is what I actually wanted - to remove them :)

Reply:
After looking at the code, I see I left semicolons all over the place